HW_7/task_3.py

```diff
@@ -0,0 +1,7 @@
+tuple_i = []
+tuple_j = []
+for i in range(1, 11):
+    tuple_i.append(i)
+    tuple_j.append(i ** 2)
+our_list = [item1 for item2 in (tuple_i, tuple_j) for item1 in item2]

```

Comment:
This needs a little rework: we get a plain list here while we need a list of tuples